dids/src/test/kotlin/web5/sdk/dids/did/BearerDidTest.kt

```diff
@@ -8,13 +8,44 @@ import org.mockito.kotlin.spy
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.whenever
 import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.dids.didcore.DidDocument
+import web5.sdk.dids.didcore.Service
+import web5.sdk.dids.methods.dht.CreateDidDhtOptions
+import web5.sdk.dids.methods.dht.DidDht
 import web5.sdk.dids.methods.jwk.DidJwk
 import kotlin.test.Test
+import kotlin.test.assertContains
 import kotlin.test.assertEquals
 import kotlin.test.assertNotNull
 
 class BearerDidTest {
 
+  @Test
+  fun `append service endpoint to existing did`() {
+    val keyManager = InMemoryKeyManager()
+    var myBearerDid = DidDht.create(keyManager, CreateDidDhtOptions(publish = true))
+
+    val existingBearerDid: BearerDid = myBearerDid
+
+    val serviceToAdd = Service.Builder()
+      .id("pfi")
+      .type("PFI")
+      .serviceEndpoint(listOf("https://example.com/"))
+      .build()
+
+    val newServiceList = existingBearerDid.document.service.orEmpty() + serviceToAdd
```

Comment:
So I think with the current impl you'll add the existing service again with this line, because you add whatever is passed in options to the existing service list? did I read the impl wrong

Reply:
hrm - I think that is my mistake, intention wasn't to only add but replace - let me add better tests and fix if needed before merging